docs/ecosystem-angular.md

```diff
@@ -3,8 +3,454 @@ id: ecosystem-angular
+[single-spa helpers](#the-single-spa-helpers) work with Angular 4.
+
+### Angular 5
+Angular 5 is supported by single-spa-angular.
+
+The [single-spa-angular schematics](#schematics) are not supported by Angular 5, so you'll have to
+follow the [steps for manual installation](#manual-installation). The
+[single-spa helpers](#the-single-spa-helpers) work with Angular 5.
+
+### Angular 6
+Angular 6 is supported by single-spa-angular.
+
+The [single-spa-angular schematics](#schematics) are not supported by Angular 6, so you'll have to
+follow the [steps for manual installation](#manual-installation). The
+[single-spa helpers](#the-single-spa-helpers) work with Angular 6.
+
+### Angular 7
+Angular 7 is supported by single-spa-angular.
+
+Both the [single-spa-angular schematics](#schematics) and the [single-spa helpers](#the-single-spa-helpers)
+work with Angular 7. Follow the [Angular CLI instructions](#angular-cli).
+
+Note that the schematics for Angular 7 use an [Angular Builder](#angular-builder) that is no longer
+used in the Angular 8 schematics.
+
+### Angular 8
+Angular 8 is supported by single-spa-angular.
+
+Both the [single-spa-angular schematics](#schematics) and the [single-spa helpers](#the-single-spa-helpers)
+work with Angular 8. Follow the [Angular CLI instructions](#angular-cli).
+
+Note that the schematics for Angular 8 [do not use the custom Angular builder](#angular-builder), but instead use
+[@angular-builders/custom-webpack](https://www.npmjs.com/package/@angular-builders/custom-webpack).
+
+## Angular CLI
+You may use Angular CLI and single-spa together with any version of Angular. However, the [Angular CLI schematics](#schematics)
+only work if you're using Angular >= 7. If you're using an older version of Angular, follow
+the [manual installation instructions](#manual-installation).
+
+### Installation
+First, create an angular application. This requires installing [Angular CLI](https://cli.angular.io/). Note that the `--prefix`
+is important so that when you have multiple angular applications their component selectors won't have the same names.
+
+```sh
+ng new my-app --routing --prefix my-app
+cd my-app
+```
+
+In the root of your Angular CLI application run the following:
+```sh
+ng add single-spa-angular@beta
+```
+
+### Schematics
+[Angular schematics](https://angular.io/guide/schematics) are processed when you run `ng add single-spa-angular@beta`.
+
+The single-spa-angular schematics perform the following tasks:
+* Install single-spa-angular.
+* Generate a `main.single-spa.ts` in your project `src/`.
+* Generate `single-spa-props.ts` in `src/single-spa/`
+* Generate `asset-url.ts` in `src/single-spa/`
+* Generate an EmptyRouteComponent in `src/app/empty-route/`, to be used in app-routing.module.ts.
+* Add an npm script `npm run build:single-spa`.
+* Add an npm script `npm run serve:single-spa`.
+* For Angular 7 only, create a new entry in the project's architect called `single-spa`, which is
```

Comment:
Just to verify, this is only for 7? Or >= 7

Reply:
Just Angular 7, because Angular 8 does it in a different way with @angular-builders/custom-webpack